src/gluonnlp/embedding/token_embedding.py

```diff
@@ -995,8 +994,9 @@ class Word2Vec(TokenEmbedding):
 
     Parameters
     ----------
-    source : str, default 'GoogleNews-vectors-negative300'
+    source : str or None, default 'GoogleNews-vectors-negative300'
         The name of the pre-trained token embedding file.
+        If source is None, construct an empty Word2Vec class object.
```

Comment:
The initializer should actually initialize the class. It seems the currently intended behavior is that users initialize with `source=None` and then call `load_w2v_binary`. How about instead renaming `load_w2v_binary` to something like `from_binary` and making it a constructor too? (ie. need to decorate it with `@classmethod`)

Reply:
Thanks for your review. Let me work on it!